src/masternodes/mn_rpc.cpp

```diff
@@ -729,10 +729,10 @@ UniValue updatetoken(const JSONRPCRequest& request) {
                        "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
                },
                RPCExamples{
-                       HelpExampleCli("updatetoken", "\"token {\\\"isDAT\\\":true}\" "
-                                                     "\"[{\\\"txid\\\":\\\"id\\\",\\\"vout\\\":0}]\"")
-                       + HelpExampleRpc("updatetoken", "\"token {\\\"isDAT\\\":true}\" "
-                                                       "\"[{\\\"txid\\\":\\\"id\\\",\\\"vout\\\":0}]\"")
+                       HelpExampleCli("updatetoken", "\"token {\"isDAT\":true}' "
```

Comment:
Single quotes needed here?

Reply:
This line should be 

`HelpExampleCli("updatetoken", " 'token {\"isDAT\":true}' "`

instead of

`HelpExampleCli("updatetoken", "\"token {\"isDAT\":true}' "`